File: C/1017_GastoDeCombustivel.c
#include <stdio.h>

int main() {

    int tempoGasto, velocidadeMedia;
    double quantidadeLitros, distancia;

    scanf("%d", &tempoGasto);
    scanf("%d", &velocidadeMedia);

    distancia = tempoGasto * velocidadeMedia;
    quantidadeLitros = distancia/12;

    printf("%.3lf\n", quantidadeLitros);

    return 0;
}

File: Python/1017_GastoDeCombustivel.py
# Entrada de dados
tempoGasto = int(input())
velocidadeMedia = int(input())

# Cálculo da distancia e da quantidade necessária de litros
distancia = tempoGasto * velocidadeMedia
quantidadeLitros = distancia / 12

# Saída de dados
print(f"{quantidadeLitros:.3f}")
